tests/phpunit/includes/functions.php

```diff
@@ -318,7 +318,8 @@ function _unhook_block_registration() {
 	remove_action( 'init', 'register_block_core_post_terms' );
 	remove_action( 'init', 'register_block_core_post_title' );
 	remove_action( 'init', 'register_block_core_query' );
-	remove_action( 'init', 'register_block_core_query_loop' );
+	remove_action( 'init', 'register_block_core_post_template' );
+	remove_action( 'init', 'gutenberg_register_legacy_query_loop_block' );
```

Comment:
Should we remove the `gutenberg` prefix here?

Reply:
https://github.com/WordPress/wordpress-develop/pull/1351#discussion_r647629755